core/src/mender-utils.c

```diff
@@ -157,8 +157,6 @@ mender_utils_deployment_status_to_string(mender_deployment_status_t deployment_s
         return "success";
     } else if (MENDER_DEPLOYMENT_STATUS_FAILURE == deployment_status) {
         return "failure";
-    } else if (MENDER_DEPLOYMENT_STATUS_ALREADY_INSTALLED == deployment_status) {
-        return "already-installed";
```

Comment:
Strictly speaking this (and the following change) we don't need to remove. It is a valid state, although the current state machine won't use it. But what if a custom Update Module wants to do some extra checks or something custom on a peripheral or something and then return no-op to the server? I would let it be.

If we really want to save the bytes, then compile it out with an ifdef for future use.

Reply:
You make a valid point. My only motivation for removing it, was that the original commit in the GO client removed it. However, the state is present in the C++ client, although never used. Furthermore, the ticket says the C++ client is the source of truth. So I see to it, that this change is removed 😉